svg/path/reftests/strokefill-001-ref.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="480" height="360"
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:html="http://www.w3.org/1999/xhtml">
+  <title>Closed, filled polygonal path with stroke-width 8</title>
+  <html:link rel="author" title="Skef Iterum" href="https://github.com/skef/"/>
+
+  <path stroke="none" fill="green" d="M87.1719 77.1719l-60 60l-1.17188 1.17285v1.65527v60v1.65527l1.17188 1.17285l60 60l1.17285 1.17188h1.65527h60h1.65527l1.17285 -1.17188l60 -60l1.17188 -1.17285v-1.65527v-60v-1.65527l-1.17188 -1.17285l-60 -60
+  l-1.17285 -1.17188h-1.65527h-60h-1.65527z"/>
+  <path stroke="none" fill="yellow" d="M34 141.655l57.6553 -57.6553h56.6895l57.6553 57.6553v56.6895l-57.6553 57.6553h-56.6895l-57.6553 -57.6553v-56.6895z"/>
+  <path stroke="black" stroke-width="2.4" fill="green" d="M87.1719 77.1719l-60 60l-1.17188 1.17285v1.65527v60v1.65527l1.17188 1.17285l60 60l1.17285 1.17188h1.65527h60h1.65527l1.17285 -1.17188l60 -60l1.17188 -1.17285v-1.65527v-60v-1.65527l-1.17188 -1.17285l-60 -60
+  l-1.17285 -1.17188h-1.65527h-60h-1.65527z"/>
+  <path stroke="black" stroke-width="2.4" fill="yellow" d="M34 141.655l57.6553 -57.6553h56.6895l57.6553 57.6553v56.6895l-57.6553 57.6553h-56.6895l-57.6553 -57.6553v-56.6895z"/>
```

Comment:
Shouldn't these two have fill="none" and be in the opposite order to match the test?

Reply:
The order of the overlines should be irrelevant -- even if they overlap the result should be "cumulative". However, the fills do need to be removed, which I've just done.

Reply:
They should still match, since the whole point is that they are supposed to be identical & we're trying to minimize any chance of arbitrary compositing differences.

But as it turns out, this isn't one of the ones currently triggering a difference.

Reply:
Alright, I matched them up.